README.md

```diff
@@ -150,6 +150,114 @@ Version:1
 $ 
 ```
 
+### List Processes Command
+```sh
+$ process-management list --help
+Usage: process management list [options]
+
+A command for listing all active process flows
+
+Options:
+  -h, --host <host>  substrate blockchain host address or FQDM, default - "localhost" (default: "localhost")
+  -p, --port <port>  specify host port number if it is not a default, default - 9944 (default: "9944")
+  --active           returns only active process flows
+  --disabled         returns only disabled process flows
+  --help             display help for command
+
+#
+# examples
+#
+
+# --active
+$ process-management list --active
+
+      retrieving all process flows from a chain...
+      options: {"host":"localhost","port":"9944","active":true}
+    
+[
+  {
+    id: '0x732e69d9bee930b67c907ac97ef0deeac8e52635c16b266071d2f42211f485a1aebdeca4745f8a62395de679a6b7332e3c6d6f636b5f706f73745f6f72646572d82c12285b5d4551f88e8f6e7eb52b8101000000',
```

Comment:
is there a way to have process id/name as regular text rather than hex and also show process versions?

Reply:
I agree, I think by default we should display regular text.  Add into another ticket a `--raw` option to display these things as Hex?

Reply:
yea, i think as dave suggested I will create a story in the back log. so far got 3 to add. Just updating integration tests.